app/src/main/java/com/atom/android/bookshop/data/model/Bill.kt

```diff
@@ -0,0 +1,41 @@
+package com.atom.android.bookshop.data.model
+
+data class Bill(
+    val id: Int,
+    val shippingMethod: ShippingMethod,
+    val address: String,
+    val note: String,
+    val phone: String,
+    val receiver: String,
+    val orderHistories: List<OrderHistory>,
+    val orderLines: List<OrderLine>,
+    val createdAt: String,
+) {
+    fun totalBill(): Double {
+        var totalBill = 0.0
+        orderLines.forEach { orderLine ->
+            totalBill += orderLine.amount * orderLine.price
+        }
+        return totalBill
+    }
```

Comment:
fun totalBill(): Double = orderLines.sumOf { it.amount * it.price }

nếu trả về Int thì toDouble :v